docs/pages/color_usage.js

```diff
@@ -0,0 +1,270 @@
+        `}
+      >
+        <ColorTile
+          description="Shopping"
+          fullTokenName="color-background-shopping"
+          name="Skycicle"
+          number={500}
+        />
+      </MainSection>
+      <MainSection
+        name="Background color"
+        description="Background colors are used behind text or applied to other elements to deliver specific messages or meanings, such as information, success, warning, and error. They are divided into two categories, base colors and weak colors. "
+      >
+        <MainSection.Subsection
+          title="Base background colors"
+          description={`
+          Use on UI elements to convey a specific status or message. Usually, they aren't used behind default text.
+          **$color-background-info-base** - Use this token as a background color to indicate neutral information.
+          **$color-background-success-base** - Use this token as a background color to indicate success.
+          **$color-background-warning-base** - Use this token as a background color to indicate warning alerts.
+          **$color-background-error-base** -  Use this tokens as a background color to indicate errors.
+        `}
+        >
+          <Flex gap={4} direction="column">
+            <Flex gap={4} wrap>
+              <ColorTile
+                description="Info base"
+                textColor="white"
+                fullTokenName="color-background-info-base"
+              />
+              <ColorTile
+                description="Success base"
+                textColor="white"
+                fullTokenName="color-background-success-base"
+              />
+            </Flex>
+            <Flex gap={4} wrap>
+              <ColorTile
+                description="Warning base"
+                textColor="white"
+                fullTokenName="color-background-warning-base"
+              />
+              <ColorTile
+                description="Error base"
+                textColor="white"
+                fullTokenName="color-background-error-base"
+              />
+            </Flex>
+          </Flex>
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Weak background colors"
+          description={`
+          Since they have a light tint, these colors can be used behind text.
+          **$color-background-info-weak** - Use this token as a background color to indicate neutral information.
+          **$color-background-success-weak** - Use this token as a background color to indicate success.
+          **$color-background-warning-weak** - Use this token as a background color to indicate warning alerts.
+          **$color-background-error-weak** -  Use this tokens as a background color to indicate errors.
+        `}
+        >
+          <Flex gap={4} direction="column">
+            <Flex gap={4} wrap>
+              <ColorTile description="Info weak" fullTokenName="color-background-info-weak" />
```

Comment:
ditto here, and also these need dark text
![Screen Shot 2022-02-16 at 11 21 01 AM](https://user-images.githubusercontent.com/12059539/154339868-638bfa2c-9fe9-437c-b8b0-737d897f9d24.png)